cdap-data-fabric/src/main/java/co/cask/cdap/data2/dataset2/InMemoryDatasetFramework.java

```diff
@@ -181,7 +181,6 @@ public synchronized void addInstance(String datasetType, Id.DatasetInstance data
     }
     DatasetDefinition def = registry.get(datasetType);
     DatasetSpecification spec = def.configure(datasetInstanceId.getId(), props);
-    LOG.info("Dataset Specification name {} and properties {}", spec.getName(), spec.getProperties());
```

Comment:
this is InMemoryDatasetFramework, the logging we discussed are in data migration. are you sure we need this log?